sunpy/net/tests/test_helioviewer.py

```diff
@@ -88,16 +89,16 @@ def test_get_jp2_header(self, client):
         assert ('fits' in header1.keys()) and ('fits' in header2.keys())
 
     @skip_glymur
+    @figure_test
     def test_download_jp2_map(self, client):
         """
         Tests getJP2Image API method with Map.
         """
-        # TODO: make this a figure test.
         filepath = client.download_jp2('2012/01/01', observatory='SOHO',
                                        instrument='MDI', measurement='continuum')
         map_ = sunpy.map.Map(filepath)
-        assert isinstance(map_, sunpy.map.GenericMap)
-        os.remove(filepath)
```

Comment:
Should we still not delete the file or at least download it into a temporary directory.

Reply:
Oh yeah, shouldn't have removed that line.